src/main/java/com/puppycrawl/tools/checkstyle/checks/modifier/ClassMemberImpliedModifierCheck.java

```diff
@@ -0,0 +1,183 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2018 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package com.puppycrawl.tools.checkstyle.checks.modifier;
+
+import com.puppycrawl.tools.checkstyle.StatelessCheck;
+import com.puppycrawl.tools.checkstyle.api.AbstractCheck;
+import com.puppycrawl.tools.checkstyle.api.DetailAST;
+import com.puppycrawl.tools.checkstyle.api.TokenTypes;
+import com.puppycrawl.tools.checkstyle.utils.ScopeUtil;
+
+/**
+ * <p>
+ * Checks for implicit modifiers on nested types in classes.
+ * </p>
+ * <p>
+ * This check is effectively the opposite of <a href="#RedundantModifier">RedundantModifier</a>.
```

Comment:
new web site should be used - checkstyle.org/.... if we end up with link like this.

@rnveach , please confirm one more time that you want link like you requested, I do not see any example in existing code base:
```
$ ag "href" src/main/ | grep "\"#"
src/main/java/com/puppycrawl/tools/checkstyle/checks/modifier/InterfaceMemberImpliedModifierCheck.java:33: * This check is effectively the opposite of <a href="#RedundantModifier">RedundantModifier</a>.
src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ParameterNameCheck.java:38: * <a href="#CatchParameterName">CatchParameterName</a>.
src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ParameterNameCheck.java:42: * <a href="#LambdaParameterName">LambdaParameterName</a>.
```

I do not remember that we agree on changing of this format. If this is smth newly noticed, lets postpone this to further Issue/PRs.

Reply:
Sorry, i am still using old URL. I updated the first post.

> I do not see any example in existing code base

I missed `ParmeterNameCheck` when we were re-organizing the Javadoc/XDoc. The reason for this request is the URL will work fine on the XDoc, but when the Javadoc is turned into HTML, there is no anchor named `RedundantModifier` so it will point to an invalid location and be worthless.

See `ParameterNameCheck` as an example: http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/checks/naming/ParameterNameCheck.html
The link to `CatchParameterName` doesn't do anything.

> I do not remember that we agree on changing of this format.

@romani If you want to drop this item since there are other bad ones, then ok.
It is not really new. We have working links for other things like TokenTypes.

See https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AtclauseOrderCheck.java#L53

Reply:
Fixing javadoc validator will require fixing the other areas too.

Lets just ignore this and I will fix it after the PR is merged.